include/drivers/entropy/nrf5_entropy.h

```diff
@@ -0,0 +1,13 @@
+/*
+ * Copyright (c) 2018 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef _NRF5_ENTROPY_H_
+#define _NRF5_ENTROPY_H_
+
+int entropy_get_entropy_thr(u8_t *buf, u16_t len);
```

Comment:
Since you are using these outside of the driver, should have a bit more comment/api docs.

Reply:
Agreed, fixed